app/helpers/notice_manager_helper.rb

```diff
@@ -0,0 +1,26 @@
+module NoticeManagerHelper
+  def notices(timeout)
+    flash.collect do |key, msg|
+     message = content_tag :span, class: 'valign-wrapper' do
+        concat msg
+        concat content_tag(:i, '', class: "icon-toast #{alert_class_finder(key)}")
+      end
```

Comment:
`end` at 7, 6 is not aligned with `message = content_tag :span, class: 'valign-wrapper' do` at 4, 5